lib/smart_proxy_openscap/openscap_initializer.rb

```diff
@@ -0,0 +1,17 @@
+require 'openscap'
+
+module Proxy::OpenSCAP
+  class OpenscapInitializer
+    include ::Proxy::Log
+
+    def start
+      logger.debug "Initializing openscap component"
+      OpenSCAP.oscap_init
+    end
+
+    def stop
```

Comment:
There's a chance that #stop is called while #start hasn't finished yet, synchronizing these calls on a mutex might be worth it (if only to prevent a potential issues on shutdown).